src/styles/consultation.scss

```
@@ -0,0 +1,51 @@
+@import "./variables.scss";
+
+.consultation {
+
+  &__wrap {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 70px;
+  }
+
+  &__text {
+    width: 412px;
+    height: 13px;
+  }
+
+  &__form-wr {
+    margin: 0 auto;
+  }
+
+  &__question {

```

Прибери цей рядок взагалі бо через нього форма криво вирівнюється на ПК екрані
```suggestion
  }

  &__question {

```